app/src/components/widgets/charts/launchExecutionAndIssueStatistics/launchExecutionAndIssuesChartLegend.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import classNames from 'classnames/bind';
+import PropTypes from 'prop-types';
+import { injectIntl, intlShape } from 'react-intl';
+import * as COLORS from 'common/constants/colors';
+import styles from '../common/legend/legend.scss';
+import { messages } from './chartUtils';
+
+const cx = classNames.bind(styles);
+
+@injectIntl
+export class Legend extends Component {
+  static propTypes = {
+    intl: intlShape.isRequired,
+    items: PropTypes.array,
+    onClick: PropTypes.func,
+    onMouseOver: PropTypes.func,
+    onMouseOut: PropTypes.func,
+  };
+
+  static defaultProps = {
+    items: [],
+    onClick: () => {},
+    onMouseOver: () => {},
+    onMouseOut: () => {},
+  };
+  onClick = ({ currentTarget }) => {
+    currentTarget.classList.toggle(cx('unchecked'));
```

Comment:
Can we put unchecked legend items in state and set this class in JSX instead of manipulating with elements directly?
Something like this:

```
state = {
  uncheckedIds: [],
};

toggleUnchecked = (id) => this.setState({uncheckedItems: /* add/remove id to this.state.uncheckedIds */});

handleClick = (id) => {
  this.toggleUnchecked(id);
  this.props.onClick(id);
}

calculateItemClassName = (id) => this.state.uncheckedIds.indexOf(id) !== -1 ? cx('unchecked') : null; // OR use .includes, but it needs a polyfill for IE 11

render() {
  // ...
  classNames={this.calculateItemClassName(item.id)}
  onClick={() => this.handleClick(item.id)}
}
```

Reply:
fixed